sources/packages/backend/apps/queues/src/main.ts

```diff
@@ -0,0 +1,41 @@
+require("../../../env_setup_apps");
+import { Queues } from "@sims/queue";
+import { createBullBoard } from "@bull-board/api";
+import { BullAdapter } from "@bull-board/api/bullAdapter";
+import { ExpressAdapter } from "@bull-board/express";
+import { NestFactory } from "@nestjs/core";
+import { Queue } from "bull";
+import { QueuesModule } from "./queues.module";
+import * as basicAuth from "express-basic-auth";
+
+async function bootstrap() {
+  const app = await NestFactory.create(QueuesModule);
+
+  // Create bull board UI dashboard for queue management.
+  const serverAdapter = new ExpressAdapter();
+  serverAdapter.setBasePath("/admin/queues");
+  const bullBoardQueues = Object.values(Queues).map((queue) => {
+    return new BullAdapter(app.get<Queue>(`BullQueue_${queue.name}`), {
+      readOnlyMode: queue.readonly,
```

Comment:
what does `readOnlyMode` do?

Reply:
It is renamed to dashboardReadOnly. This property is decides if the queue has to be displayed in bullboard ui as read only.